docs/updating/2_merge_composer.md

```diff
@@ -0,0 +1,36 @@
+# 2. Merge composer.json
+
+## Manual merging
+
+!!! note "Sorted packages since 2.4 changes"
+
+    Since 2.4 packages in `composer.json` are sorted, which means more conflicts when updating to 2.4, but far fewer conflicts in the future. This is controlled by [sort-packages](https://getcomposer.org/doc/06-config.md#sort-packages) config in `composer.json`.
```

Comment:
```suggestion
    Because the 2.4 packages in `composer.json` are sorted, there will be more conflicts when updating to 2.4, but far fewer conflicts in the future. This is controlled by [sort-packages](https://getcomposer.org/doc/06-config.md#sort-packages) config in `composer.json`.
```